infra/recipes/docker-compose/common/aws-services.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.3"
+services:
+  minioS3:
+    image: minio/minio
+    environment:
+      - MINIO_ROOT_USER=admin
+      - MINIO_ROOT_PASSWORD=password
+    ports:
+      - 9871:9001
+      - 9870:9000
+    command: [ "server", "/data", "--console-address", ":9001" ]
+  mc:
```

Comment:
What does mc stand for?

Reply:
MinIO client? Can we add a comment describing that?

Reply:
Added a comment and also renamed the container